article/RNH_AR_EG_SSTJ_EP_MMC-2016.md

```diff
@@ -98,62 +110,86 @@ in the equations describing the first proposed algorithm. Firstly, the free-wate
+of the model weights ($\omega_i=s_i$):
+
+$$F_{WLS} = \frac{1}{2} \sum_{i=1}^{m} \left
+ [ \omega_i \left ( y_{i} -\sum_{j=1}^{7}W_{ij}\gamma_{j}\right ) \right ]^{2}$$ {#eq:6}
 
-$$F_{NLS} = \frac{1}{2} \sum_{m}^{i=1} \left
- [s_{i} - S_{0} f\exp(-\sum_{j=2}^{4}W_{ij}D_{iso})
-- (1-f)\exp(-\sum_{j=1}^{7}W_{ij}\gamma_{j})\right ]^{2}$$ {#eq:3}
+when evaluating which ($f$, $D_tissue$) pair is associated with smaller residuals the NLS
+objective function is used instead:
 
-Similarly to the original article [@Hoy2014-lk,], the WLS procedure is only used here
+$$F_{NLS} = \frac{1}{2} \sum_{i=1}^{m} \left
+ [s_{i} - S_{0} f\exp(-b_iD_{iso})
+- (1-f)\exp(-\sum_{j=1}^{7}W_{ij}\gamma_{j})\right ]^{2}$$ {#eq:7}
+
+Similarly to the original article [@Hoy2014-lk], these procedure is only used
 to obtain the intial guess for the free water elimination parameters, which were then
-used to initialize the non-linear convergence procedure (see below).
+used to initialize a fwDTI model non-linear convergence solver (see below).
 
-**Non-Linear Least Square Solution (NLS)**. As suggested by Hoy and colleagues
-[@Hoy2014-lk,], the initial guess for the non-linear convergence
-procedure was set to the values estimated from the WLS approach. To improve the speed of
-computation, instead of using the modified Newton's algorithm proposed in the original article,
+**Non-Linear Least Square Solution (NLS)**. To improve
+computation speed, instead of using the modified Newton's algorithm proposed in the original article,
 the non-linear convergence was done using Scipy's wrapped modified Levenberg-Marquardt algorithm
-(function `scipy.optimize.leastsq` of Scipy http://scipy.org/). To constrain the
-model parameters to within a plausible range, the free water volume fraction $f$ was
-converted to $f_t = \arcsin (2f-1) + \pi / 2$. To compare the robustness of the
-techniques with and without this constraint, the free water volume fraction transformation was implemented
-as an optional feature that can be controlled through user-provided arguments. In addition to
-the `scipy.optimize.leastsq` function, a more recently implemented version of Scipy's optimization function
-`scipy.optimize.least_square` (available as of Scipy's version 0.17) was also tested.
+(function `scipy.optimize.leastsq` of Scipy http://scipy.org/).
+
+To constrain the model parameters to within a plausically ranges, some variable transformations
+can be applied to the non-linear objective function. These were implemented as optional features
+that can be controlled through user-provided arguments. To restrict the range of the volume fraction
+to values between 0 and 1, the variable $f$ in equation 7 can be replaced by $$\sin(f_t - \pi/2)/2+1/2$$
+and non-linear convergence is performed as function of $f_t$. To ensure that the diffusion tensor is
+positive definite, diffusion parameters can be converted to the Cholesky decomposition elements according [@Koay2006-zo]
```

Comment:
=> "... according to [@Koay2006-zo]." (including a period at the end of the sentence!)